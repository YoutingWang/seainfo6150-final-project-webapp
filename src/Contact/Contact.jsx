import React from "react";
import Navigation from "../NavigationBar/Navigation";
import styles from "./Contact.module.css";
import { useHistory } from "react-router-dom";

const Contact = () => {
    const history = useHistory();
    function handleClick() {
        history.push("/");
    }
    return (
        <div>
            <Navigation/>
            <h2 className={styles.h2}>Contact Us</h2>
            <p className={styles.text}>If you have any questions, please leave your name and contact information. We will contact you later!</p>
            <form className={styles.form}>
            <label className={styles.text} htmlFor="name">Name</label>
            <input type="text" name="name" id="name" placeholder="Name" /><br/>
            
            <label className={styles.text} htmlFor="email">Email</label>
            <input type = "email" required = "required" name = "Email" placeholder = "Email" id = "email"/> <br/>

            <label className={styles.text} htmlFor="phonenumber">Phone Number</label>
            <input type = "tel" required = "required" name = "PhoneNumber" placeholder = "PhoneNumber" id = "phonenumber"/><br/>

            <label className={styles.text} htmlFor="City">Which city do you live in? </label>
            <select name="City" id="City">
              <option value="San Jose">San Jose</option>
              <option value="LA">LA</option>
              <option value="Seattle">Seattle</option>
              <option value="New York">New York</option>
            </select>
            <br/>

            <input className={styles.button} type="submit" value="Submit" onClick={handleClick} />
        
          </form>
        </div>
    );
};

export default Contact;