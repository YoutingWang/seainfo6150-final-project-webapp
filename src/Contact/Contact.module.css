.h2{
    text-align: center;
    font-family: 'Raleway', sans-serif;
    font-size: 30px;
}
.text{
    text-align: center;
    font-family: 'Nunito', sans-serif;
    font-size: 18px;
    color: black;
    border-radius: 30px;
}
.form{
    color: black;
    font-family: 'Nunito', sans-serif;
    text-align: center;
    margin-top: 8px;
}
.button{
    font-family: 'Nunito', sans-serif;
    position: relative;
    margin-top: 10px;
    color: black;
    border-radius: 20px;
}