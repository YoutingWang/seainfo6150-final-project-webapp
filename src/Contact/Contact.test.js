import React from "react";
import {
    render
} from "@testing-library/react";
import Contact from "./Contact.jsx";

it("renders the home component correctly", () => {
    const {
        container
    } = render( < Contact / > );
    expect(container).toMatchSnapshot();
});