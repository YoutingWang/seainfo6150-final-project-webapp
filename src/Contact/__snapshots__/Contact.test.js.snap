// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component correctly 1`] = `
<div>
  <div>
    <div
      class="header"
    >
      <div
        class="left"
      >
        <img
          alt="Logo"
          class="logo"
          src="Logo.jpg"
        />
        <h1
          class="title"
        >
          Yummy Restaurants
        </h1>
      </div>
      <div
        class="right"
      >
        <button
          type="button"
        >
          Log in
        </button>
        <button
          type="button"
        >
          Sign up
        </button>
      </div>
    </div>
    <h2
      class="h2"
    >
      Contact Us
    </h2>
    <p
      class="text"
    >
      If you have any questions, please leave your name and contact information. We will contact you later!
    </p>
    <form
      class="form"
    >
      <label
        class="text"
        for="name"
      >
        Name
      </label>
      <input
        id="name"
        name="name"
        placeholder="Name"
        type="text"
      />
      <br />
      <label
        class="text"
        for="email"
      >
        Email
      </label>
      <input
        id="email"
        name="Email"
        placeholder="Email"
        required=""
        type="email"
      />
       
      <br />
      <label
        class="text"
        for="phonenumber"
      >
        Phone Number
      </label>
      <input
        id="phonenumber"
        name="PhoneNumber"
        placeholder="PhoneNumber"
        required=""
        type="tel"
      />
      <br />
      <label
        class="text"
        for="City"
      >
        Which city do you live in? 
      </label>
      <select
        id="City"
        name="City"
      >
        <option
          value="San Jose"
        >
          San Jose
        </option>
        <option
          value="LA"
        >
          LA
        </option>
        <option
          value="Seattle"
        >
          Seattle
        </option>
        <option
          value="New York"
        >
          New York
        </option>
      </select>
      <br />
      <input
        class="button"
        type="submit"
        value="Submit"
      />
    </form>
  </div>
</div>
`;
